kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/DocumentationUrls.java

```diff
@@ -0,0 +1,247 @@
+            return url.replace(placeHolder, majorFxVersion);
+        }
+        return url;
+    }
+
+    public String getJavadocHome() {
+        return getAsMandatoryValue(DocumentationItem.JAVADOC_HOME);
+    }
+
+    public String getOracleDocumentation() {
+        return getAsMandatoryValue(DocumentationItem.ORACLE_DOCUMENTATION);
+    }
+
+    public String getGluonJavadocHome() {
+        return getAsMandatoryValue(DocumentationItem.GLUON_JAVADOC_HOME);
+    }
+
+    public String getOpenjfxGettingStarted() {
+        return getAsMandatoryValue(DocumentationItem.OPENJFX_GETTING_STARTED);
+    }
+
+    public String getOpenjfxJavadocHome() {
+        return getAsMandatoryValue(DocumentationItem.OPENJFX_JAVADOC_HOME);
+    }
+
+    public String getOpenjfxCssReference() {
+        return getAsMandatoryValue(DocumentationItem.OPENJFX_CSS_REFERENCE);
+    }
+
+    public String getOpenjfxFxmlReference() {
+        return getAsMandatoryValue(DocumentationItem.OPENJFX_FXML_REFERENCE);
+    }
+
+    public String getGluonScenebuilderHome() {
+        return getAsMandatoryValue(DocumentationItem.GLUON_SCENEBUILDER_HOME);
+    }
+
+    public String getGettingStartedWithKotlin() {
+        return getAsMandatoryValue(DocumentationItem.KOTLIN_GETTING_STARTED);
+    }
+
+    String getAsMandatoryValue(DocumentationItem item) {
+        String url = configuredUrls.get(item);
+        assert url != null;
+        return url;
+    }
+
+    public Optional<String> getOptionalUrl(DocumentationItem item) {
+        return Optional.ofNullable(configuredUrls.get(item));
+    }
+
+    /**
+     * The enum item name is used as properties key, therefore all "_" are replaced
+     * with "." and the string is turned lowercase.
+     * 
+     * For some items defaults are defined as those are required in various places
+     * inside SceneBuilder.
+     * 
+     * The items without default are optional. So removing them from the resource
+     * will also lead to removal of these items from GUI.
+     * 
+     */
+    public enum DocumentationItem {
+        JAVADOC_HOME("https://openjfx.io/javadoc/11/"),
+        ORACLE_DOCUMENTATION("https://docs.oracle.com/javafx/index.html"),
+        GLUON_JAVADOC_HOME("https://docs.gluonhq.com/charm/javadoc/latest/"),
+        GLUON_SCENEBUILDER_HOME("https://gluonhq.com/products/scene-builder/"),
+
+        OPENJFX_GETTING_STARTED("https://openjfx.io/openjfx-docs/"),
+        OPENJFX_JAVADOC_HOME("https://openjfx.io/javadoc/16/"),
+        OPENJFX_CSS_REFERENCE("https://openjfx.io/javadoc/16/javafx.graphics/javafx/scene/doc-files/cssref.html"),
+        OPENJFX_FXML_REFERENCE("https://openjfx.io/javadoc/16/javafx.fxml/javafx/fxml/doc-files/introduction_to_fxml.html"),
+
+        KOTLIN_GETTING_STARTED("https://kotlinlang.org/docs/getting-started.html"),
+
+        GLUON_SCENEBUILDER_CONTRIBUTE;
+
+        private final String defaultUrl;
+        
+        private DocumentationItem() {
+            this(null);
```

Comment:
I'm sure we can avoid this null. Is it just because of `GLUON_SCENEBUILDER_CONTRIBUTE`, then I'd say we just add the github link here.

Reply:
I've removed this, the setup has now changed as the properties file also no longer exists.